### PR TITLE
Rails 7

### PR DESCRIPTION
**Pull request by StevenLeighton21**

### CODE REVIEW

File: config/environments/production.rb [hunk 1]
```diff
@@ -56,40 +58,19 @@
   # the I18n.default_locale when a translation cannot be found).
   config.i18n.fallbacks = true
 
-  # Send deprecation notices to registered listeners.
-  config.active_support.deprecation = :notify
+  # Don't log any deprecations.
+  config.active_support.report_deprecations = false
```
Comment (StevenLeighton21) [inline config/environments/production.rb:62]: I think I'll revert this change

File: config/initializers/assets.rb [hunk 2]
```diff
@@ -1,12 +1,10 @@
 # Be sure to restart your server when you modify this file.
 
 # Version of your assets, change this if you want to expire all your assets.
-Rails.application.config.assets.version = '1.0'
+Rails.application.config.assets.version = "1.0"
 
 # Add additional assets to the asset load path.
 # Rails.application.config.assets.paths << Emoji.images_path
-# Add Yarn node_modules folder to the asset load path.
-Rails.application.config.assets.paths << Rails.root.join('node_modules')
```
Comment (StevenLeighton21) [inline config/initializers/assets.rb:9]: Have not noticed issues with this, but interested if there's a reason this was here/removed that isn't driven by the removal of sprockets

Comment (chrispymm) [reply config/initializers/assets.rb:9]: Pretty sure this would only impact sprockets, which we don't use so it should be fine

**Review by H3ll3m4 — APPROVED** [review]
🚀

**Review by chrispymm — APPROVED** [review]

**Pull request merged**